_includes/docs_menu.html

```diff
@@ -7,7 +7,8 @@
     <li>Basic Concepts</li>
     <li><a href="{{ site.baseurl }}/docs/design">Design Considerations</a></li>
     <li><a href="{{ site.baseurl }}/docs/dialect">Create a Dialect Definition (TDD)</a></li>
-    <li><a href="{{ site.baseurl }}/docs/ui">Build the Connection Dialog</a></li>
+    <li><a href="{{ site.baseurl }}/docs/ui">Build the Connection Dialog with Connection Dialog v1</a></li>
```

Comment:
If the sidebar starts to look weird with long titles we could try "Build the Connection Dialog with 1.0"/"... with 2.0".  We can always iterate.